WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -494,7 +494,23 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
     }
 
     func cell(_ cell: UITableViewCell, handleTrashPost post: AbstractPost) {
-        deletePost(post)
+        if (post.status == "trash") {
+            weak var weakSelf = self
+
+            let cancelText = NSLocalizedString("Cancel", comment: "Cancels an Action")
+            let deleteText = NSLocalizedString("OK", comment: "Deletes post permanently")
+            let messageText = NSLocalizedString("Delete Permanently", comment: "Deletes post permanently")
```

Comment:
I'd perhaps match the text used by Calypso here, to make it a bit more informative: "Delete this post permanently?"